CHANGELOG.md

```diff
@@ -8,9 +8,14 @@ Guide: https://keepachangelog.com/en/1.0.0/
 
 <!-- Add changes for active work here -->
 
+## 2.5.1
```

Comment:
I was expecting the 2.6.0-beta.1 release with Common 24.8.0-beta.1. Do you plan to release both, v2.5.1 and v2.6.0-beta.1?

Reply:
Yes